### PR TITLE
chore: add node 8 and 10 in ci

### PR DESCRIPTION
**Pull request by jingsam**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
+  - "8"
   - "10"
-  - "node"
```
Comment (mourner) [inline .travis.yml]: Don't remove this line — it signifies "latest stable", currently Node v12.

**Comment by jingsam** [comment]
I think it would be better to explicitly specify node versions than an unibigous one. Shall I add node 12 to ci?

**Comment by mourner** [comment]
@jingsam I prefer to have implicit latest among Travis Node versions in all my projects.

**Comment by jingsam** [comment]
@mourner I have add node latest in ci tests.

**Comment by mourner** [comment]
Thanks!

**Pull request merged**